_includes/forms/input.html

```diff
@@ -0,0 +1,18 @@
+{% assign name = include.name | default: "input" %}
+{% assign type = include.type | default: "text" %}
+{% assign label = include.label | default: {{ name }} %}
```

Comment:
```suggestion
{% assign label = include.label | default: name %}
```

Don't need `{{ }}` inside of `{% %}`